src/utils/message_builder.rs

```diff
@@ -723,6 +727,190 @@ impl fmt::Display for MessageBuilder {
     }
 }
 
+
+/// Content modifier enum
+///
+/// Provides an enum of formatting modifiers for a string, for combination with
+/// string types and Content types.
+///
+/// # Examples
+///
+/// Create a new Content type which describes a bold-italic "text":
+///
+/// ```rust
+/// use serenity::utils::{Bold, Italic, Content};
+/// let content: Content = Bold + Italic + "text";
+/// ```
+pub enum ContentModifier {
+  Italic,
+  Bold,
+  Strikethrough,
+  Code,
+  Underline
+}
+
+/// Describes formatting on string content
+#[derive(Default, Clone)]
+pub struct Content {
+  italic: bool,
+  bold: bool,
+  strikethrough: bool,
+  pub inner: String,
+  code: bool,
+  underline: bool
+}
+
+impl Add<String> for Content {
+  type Output = Content;
+  fn add(self, rhs: String) -> Content {
+    let mut nc = self.clone();
+    nc.inner = nc.inner + &rhs;
+    nc
```

Comment:
An empty line before the return (as well as the others) please -- it helps with readability.